package.json

```
@@ -8,17 +8,18 @@
     "dist"
   ],
   "scripts": {
-    "build": "webpack --config webpack.config.js && npm run api-report",
+    "build": "webpack --config webpack.config.js",
```

I think `rollup` replaces the previous `build` command. The current `build` is not making a full d.ts.

How about removing `rollup` and using it as the `build`?

```suggestion
    "build": "webpack --config webpack.config.js && npm run api-report && npm run prune",
```